Trimble.Modus.Components/Controls/TMDataGrid/DataGridColumn.cs

```diff
@@ -0,0 +1,256 @@
+    private ColumnDefinition? _columnDefinition;
+    private readonly ColumnDefinition _invisibleColumnDefinition = new(0);
+    private readonly WeakEventManager _sizeChangedEventManager = new();
+
+    #endregion Fields
+    internal DataGrid? DataGrid { get; set; }
+
+    public DataGridColumn()
+    {
+        HeaderLabel = new Label { Margin = new Thickness(0,0,0,0)};
+        SortingIcon = new Image {Source = ImageConstants.CaretDownImage};
+        SortingIconContainer = new ContentView
+        {
+            IsVisible = false,
+            Content = SortingIcon,
+            HorizontalOptions = LayoutOptions.Center,
+            VerticalOptions = LayoutOptions.Center,
+        };
+    }
+
+    #region Events
+
+    public event EventHandler SizeChanged
+    {
+        add => _sizeChangedEventManager.AddEventHandler(value);
+        remove => _sizeChangedEventManager.RemoveEventHandler(value);
+    }
+
+    #endregion Events
+
+    #region Bindable Properties
+
+    public static readonly BindableProperty WidthProperty = BindableProperty.Create(nameof(Width), typeof(GridLength), typeof(DataGridColumn), GridLength.Star, propertyChanged: OnWidthPropertyChanged);
+    public static readonly BindableProperty TitleProperty = BindableProperty.Create(nameof(Title), typeof(string), typeof(DataGridColumn), string.Empty, propertyChanged: (b, _, n) => ((DataGridColumn)b).HeaderLabel.Text = (string)n);
+    public static readonly BindableProperty PropertyNameProperty = BindableProperty.Create(nameof(PropertyName), typeof(string), typeof(DataGridColumn));
+    public static readonly BindableProperty IsVisibleProperty = BindableProperty.Create(nameof(IsVisible), typeof(bool), typeof(DataGridColumn), true, propertyChanged: OnIsVisiblePropertyChanged);
+    public static readonly BindableProperty CellTemplateProperty = BindableProperty.Create(nameof(CellTemplate), typeof(DataTemplate), typeof(DataGridColumn));
+    public static readonly BindableProperty LineBreakModeProperty = BindableProperty.Create(nameof(LineBreakMode), typeof(LineBreakMode), typeof(DataGridColumn), LineBreakMode.WordWrap);
+    public static readonly BindableProperty HorizontalContentAlignmentProperty = BindableProperty.Create(nameof(HorizontalContentAlignment), typeof(LayoutOptions), typeof(DataGridColumn), LayoutOptions.Center);
+    public static readonly BindableProperty VerticalContentAlignmentProperty = BindableProperty.Create(nameof(VerticalContentAlignment), typeof(LayoutOptions), typeof(DataGridColumn), LayoutOptions.Center);
+    public static readonly BindableProperty SortingEnabledProperty = BindableProperty.Create(nameof(SortingEnabled), typeof(bool), typeof(DataGridColumn), true);
+    public static readonly BindableProperty FormattedTitleProperty = BindableProperty.Create(nameof(FormattedTitle), typeof(FormattedString), typeof(DataGridColumn), propertyChanged: (b, _, n) => ((DataGridColumn)b).HeaderLabel.FormattedText = (FormattedString)n);
```

Comment:
Is the FormattedTitle used anywhere?

Reply:
Its not used by us, its a property to set formatted header title for columns.

Reply:
I guess this is removed now?

Reply:
This property is removed